ajax/libs/pagecss/package.json

```diff
@@ -0,0 +1,32 @@
+  {
+    "name": "pagecss",
+    "version": "0.3.0",
+    "filename": "page.min.css",
+    "author": "Andrew Chilton",
+    "keywords": [
+      "css",
+      "pagecss",
+      "front end"
+    ],
+    "description": "Beautiful CSS for single page sites - simple all the way up!",
+    "repository": {
+      "type": "git",
+      "url": "git://github.com/chilts/pagecss",
+      "dependencies": {
+        "node-sass": "^4.5.3"
+      }
+    },
+    "autoupdate": {
+      "source": "git",
+      "target": "git://github.com/chilts/pagecss.git",
+      "fileMap": [
+	      {
+         "basePath": "dist",
+  	  "files": [
+            "*/"
```

Comment:
@dakshshah96 Thanks for the link! Yeah globs are kinda new to me.
@PeterDaveHello I got that all fixed. I'm just hoping the indent is okay. The reason I keep switching it is because in the CI testing it says "use two space indent", however the JSONLint linter is giving me a different format after I lint it.
Almost there 🥇